frontend-guide.md

```diff
@@ -0,0 +1,333 @@
+# Front Row Frontend
+
+This document is meant to give a broad overview of the state of the frontend codebase of FrontRow.
+
+## Applications
+
+All the frontend codebase exist in [`megarepo/frontend`](https://github.com/frontrowed/megarepo/tree/master/frontend).
+
+The frontend is split in 3 different applications:
+  - [Teacher Dashboard](https://classroom.frontrowed.com)
+  - [Student Dashboard](https://student.frontrowed.com)
+  - [School Dashboard](https://school.frontrowed.com)
+
+All applications use the same stack, share the same tools, build processes, guidelines, architecture, and produce the same type of output.
```

Comment:
"All applications use the same stack; share the same tools, build processes, guidelines, and architecture; and produce the same type of output"